### ISSUE TITLE
BUG: "label" gets overwritten when it retrieves a PDB from the PDB

### ISSUE THREAD

**Issue by brettbarbaro** [corredD/mesoscope#18]

**Comment by corredD** [comment]
Thats right I should test if the label is already present and not overwrite it then.

**Comment by corredD** [comment]
Although I am not sure about it, there is an issue with the table view which doesnt update so sometimes what you see is actually not the actual data, until you move something that will force a redraw.
